src/dmd/mars.d

```diff
@@ -70,6 +70,25 @@ private void logo()
     printf("DMD%llu D Compiler %s\n%s %s\n", cast(ulong)size_t.sizeof * 8, global._version, global.copyright, global.written);
 }
 
+/**
+Print DMD's logo with more debug information and error-reporting pointers.
+
+Params:
+    stream = output stream to print the information on
+*/
+extern(C) void internalFailureLogo(FILE* stream)
```

Comment:
The backend was printing to `stdout` and I didn't want to change this behavior in this PR.